### ISSUE TITLE
Annovar db not downloading

### ISSUE THREAD

**Issue by silviatgl** [a-xavier/tapes#8]
Dear all, 

I just installed tapes, and when I run the command:

```
python3 tapes.py db -s -A /path/to/annovar/

```
Throws an error (see photo)

Any help?

thanks!

![WhatsApp Image 2020-11-03 at 16 54 15](https://user-images.githubusercontent.com/63717815/98029131-5d160900-1e0f-11eb-9be3-0379cceb6592.jpeg)

**Comment by FedikZe** [comment]
Hi, 
I have same problem..
 python3 tapes.py db -s -A /home/fil/annovar/

NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg19_avdblist.txt.gz ... Failed
WARNING: Some files cannot be downloaded, including http://www.openbioinformatics.org/annovar/download/hg19_avdblist.txt.gz
NOTICE: Web-based checking to see whether ANNOVAR new version is available ... Done
NOTICE: Downloading annotation database http://www.openbioinformatics.org/annovar/download/hg38_avdblist.txt.gz ... Failed
WARNING: Some files cannot be downloaded, including http://www.openbioinformatics.org/annovar/download/hg38_avdblist.txt.gz
Traceback (most recent call last):
  File "tapes.py", line 421, in <module>
    tf.check_online_annovar_dbs(annovar_path)
  File "/home/fil/tapes/src/t_func.py", line 888, in check_online_annovar_dbs
    with open(outfile_hg19, 'r') as file:
FileNotFoundError: [Errno 2] No such file or directory: '/home/fil/annovar/hg19_avdblist.txt'

I think there is a problem with a link to db, because now is annovar.openbioinformatics....

**Comment by a-xavier** [comment]
Hi,

I had a look and I was able to replicate the error.

It looks like ANNOVAR itself does not produce the expected output ```hg19_avdblist.txt``` when using the command 
```perl annotate_variation.pl -buildver hg19 -downdb -webfrom annovar avdblist ./```.

The problem comes from ANNOVAR itself so we can expect it to be fixed pretty soon.
Using any other argument (such as a real database such as ```clinvar_20190305``` instead of ```avdblist```) does work and downloads the proper database.

As a workaround, I attached 2 relatively recent avdblist files. 
[hg19_avdblist.txt](https://github.com/a-xavier/tapes/files/5507866/hg19_avdblist.txt)
[hg38_avdblist.txt](https://github.com/a-xavier/tapes/files/5507867/hg38_avdblist.txt)
They need to be placed in the ANNOVAR folder (where ```annotate_variation.pl``` is located)

When downloading the list of ANNOVAR database fails, it will use those instead.

I will investigate a better  solution soon.

Kind regards,
Alex
